tasks/log_groups.yml

```diff
@@ -1,14 +1,15 @@
 ---
-
 - name: Ensure log groups {{ item.log_group_name }} exist 
   cloudwatchlogs_log_group:
     log_group_name: "{{ item.log_group_name }}"
-    retention: "{{item.log_retention|int}}"
-    tags: "{{item.log_group_tags}}"
-    aws_access_key: "{{ deploy_aws_access_key | default(aws_access_key | default(lookup('env','AWS_ACCESS_KEY_ID'))) }}"
-    aws_secret_key: "{{ deploy_aws_secret_key | default(aws_secret_key | default(lookup('env','AWS_SECRET_ACCESS_KEY'))) }}"
-    region: "{{ aws_region }}"
+    retention: "{{ item.log_retention|int }}"
+    tags: "{{ item.log_group_tags }}"
+    aws_access_key: "{{ deploy_aws_access_key | default(lookup('env','AWS_ACCESS_KEY_ID')) }}"
+    aws_secret_key: "{{ deploy_aws_secret_key | default(lookup('env','AWS_ACCESS_KEY_ID')) }}"
+    security_token: "{{ security_token | default(omit) }}"
```

Comment:
I've changed this back to use `deploy_aws_access_key` and `deploy_aws_secret_key` by default but these should be treated as unique and separate from wg-ansible credentials and so set explicitly in wg-ansible where it makes sense. That way if later we change the name of that credential in wg-ansible we won't break this role!

Reply:
I think this is ok. The only other thing you may need to change (other than the other mass of similar bits of code) is https://github.com/Webgains/wg-ansible/blob/fe66982103914f9a096f3fe7a1752072f58a204f/Vagrantfile#L172 that should be changed to set `deploy_aws_access_key` etc. 

However I still think it would just be better to leave as the original for now as that is 'working'. Then tackle the issue as a whole. Ultimately these lines just need removing with the assumption that the AWS SDK is configured with the correct credentials outside of Ansible.

Reply:
The original was no working sadly. This change had to be made to support assumable roles.

Reply:
Adding the `security_token` is what was wrong with role as you need all three vars, but the `aws_access_key` and `aws_secret_key` should be fine as they where, and as commented elsewhere `security_token` should follow a similar model until removed

Reply:
So are you saying the `security_token` should be used while assuming a role and while only using a user?